.travis.yml

```diff
@@ -137,7 +137,8 @@ before_script:
   - ((phantomjs --webdriver=4422 2>/dev/null >/dev/null &) && echo "PhantomJS started succesfully") || (echo "Failed to start PhantomJS!" ; exit 1)
   - #PERL5OPT="$PERL5OPT $HARNESS_PERL_SWITCHES" plackup -Ilib --port 5001 --server HTTP::Server::PSGI tools/starman.psgi >/dev/null 2>/dev/null &
   - PERL5OPT="$PERL5OPT $HARNESS_PERL_SWITCHES" plackup -Ilib --port 5001 --server HTTP::Server::PSGI tools/starman.psgi --access_log /tmp/plackup-access.log 2> /tmp/plackup-error.log &
-  - #sleep 5  # give starman a chance to start # we now have better handling for this in Is_LSMB_running.sh
+  - sleep 5  # give starman a chance to start # we now have better handling for this in Is_LSMB_running.sh
+  - cat /tmp/plackup-error.log
```

Comment:
@einhverfr can you please drop in to #ledgersmb to discuss this.
The sleep should no longer be required due to changes in "Is_LSMB_Running.sh" 

However, the failure you saw in testing didn't dump logfiles as expected so we need to investigate that